### PR TITLE
Show reviewers on the dashboard

### LINKED ISSUE #351: See items needing review
As a collection reviewer, I should be able to see items waiting for review in the collections that I am a reviewer of.

### PR DESCRIPTION
**Pull request by jcoyne**
## Why was this change made?

Fixes #351

## How was this change tested?
Screen Shot 2020-11-10 at 10 33 45 AM

### CODE REVIEW

File: app/controllers/dashboards_controller.rb [hunk 1]
```diff
@@ -10,7 +10,10 @@ def show
     authorize! :dashboard
     @presenter = DashboardPresenter.new(
       collections: authorized_scope(Collection.all),
-      drafts: Work.where(state: 'first_draft', depositor: current_user)
+      approvals: Work.with_state(:pending_approval)
+                      .joins(collection: :reviewers)
+                      .where('reviewers.user_id' => current_user),
```
Comment (mjgiarlo) [inline app/controllers/dashboards_controller.rb:15]: I'm a bit surprised Rubocop isn't opinionated about these two lines being indented one character. I guess if Rubocop doesn't care... meh? :robot:  :police_officer:

File: app/models/work.rb [hunk 2]
```diff
@@ -31,6 +31,10 @@ class Work < ApplicationRecord
       transition first_draft: :deposited, version_draft: :deposited
     end
 
+    event :submit_for_review do
+      transition first_draft: :pending_approval
```
Comment (mjgiarlo) [inline app/models/work.rb:35]: And we'll add `version_draft: :pending_approval` later when we work on versioning, I assume

Comment (mjgiarlo) [inline app/models/work.rb:35]: `pending_approval` or `pending_review` or :man_shrugging: ?

File: app/views/works/show.html.erb [hunk 3]
```diff
@@ -47,7 +47,7 @@
         </tr>
         <tr>
           <th scope="row">Version details</th>
-          <td><%= @work.version %>></td>
+          <td><%= @work.version %></td>
```
Comment (mjgiarlo) [inline app/views/works/show.html.erb:50]: :1st_place_medal:

**Review by mjgiarlo — APPROVED** [review]
I have a couple minor, non-blocking questions. :tada:

File: db/structure.sql [hunk 4]
```diff
@@ -354,6 +353,16 @@ CREATE SEQUENCE public.related_works_id_seq
 ALTER SEQUENCE public.related_works_id_seq OWNED BY public.related_works.id;
 
 
+--
+-- Name: reviewers; Type: TABLE; Schema: public; Owner: -
+--
+
+CREATE TABLE public.reviewers (
```
Comment (jmartin-sul) [inline db/structure.sql:360]: 🎉

**Comment by jmartin-sul** [comment]
i like the switch to the join table for reviewers.

File: app/models/work.rb (hunk 2 above)
Comment (jcoyne) [reply app/models/work.rb:35]: yes.

Comment (jcoyne) [reply app/models/work.rb:35]: "Pending approval" was the language used in the mockups, so I used that.

Comment (mjgiarlo) [reply app/models/work.rb:35]: Good call!

**Pull request merged**